shepherd/coding_challenge_problems.py

```diff
@@ -0,0 +1,217 @@
+import subprocess
+import doctest
+import os
+from utils import *
+import importlib
+import re
+
+# TODO: replace with this year's challenges
```

Comment:
should probably clean this out with a more skeleton version

Reply:
done